src/openassetio-python/tests/package/hostApi/test_manager.py

```diff
@@ -240,6 +240,54 @@ def test_when_entity_ref_prefix_type_invalid_then_debug_log_printed(
         )
 
 
+manager_capabilities = [
+    (Manager.Capability.kStatefulContexts, ManagerInterface.Capability.kStatefulContexts),
+    (Manager.Capability.kCustomTerminology, ManagerInterface.Capability.kCustomTerminology),
+    (Manager.Capability.kResolution, ManagerInterface.Capability.kResolution),
+    (Manager.Capability.kPublishing, ManagerInterface.Capability.kPublishing),
+    (Manager.Capability.kRelationshipQueries, ManagerInterface.Capability.kRelationshipQueries),
+    (Manager.Capability.kExistenceQueries, ManagerInterface.Capability.kExistenceQueries),
+    (
+        Manager.Capability.kDefaultEntityReferences,
+        ManagerInterface.Capability.kDefaultEntityReferences,
+    ),
+]
+
+
+class Test_Manager_Capability:
+    @pytest.mark.parametrize(
+        "manager_capability,managerinterface_capability", manager_capabilities
+    )
+    def test_values_match_managerinterface(self, manager_capability, managerinterface_capability):
+        assert manager_capability.value == managerinterface_capability.value
+
```

Comment:
[fix] Needs a num values test

Reply:
Do you mean to check the numeric values of all the enums, like we do for `ManagerInterface.Capability`?  

I'm not sure that's necessary, since the only guarantee (currently) is that the numeric values match the `ManagerInterface.Capability` values, which are then tested independently...?

Reply:
Sorry, just to check how many there are, we do [here](https://github.com/OpenAssetIO/OpenAssetIO/pull/1126/files#diff-05562a713140e71a048205b438edf81769892ade3d32b2d004c652281fdc6395R77)?

Reply:
Ah yes, good idea

Reply:
Added in c6ed7dbf